python/tests/res/enkf/test_enkf.py

```diff
@@ -90,7 +90,7 @@ def test_invalid_res_config(self):
         with TestAreaContext("enkf_test") as work_area:
             with self.assertRaises(TypeError):
                 work_area.copy_directory(self.case_directory)
-                main = EnKFMain("simple_config/minimum_config", res_config=self)
+                main = EnKFMain(res_config=self)
```

Comment:
This looks *really* weird - `self` should be an instance of `ExtendedTestCase`??